classes/cargo.bbclass

```diff
@@ -89,6 +89,7 @@ oe_cargo_fix_env () {
 	export HOST_CC="${RUST_BUILD_CC}"
 	export HOST_CFLAGS="${RUST_BUILD_CFLAGS}"
 	export HOST_AR="${BUILD_AR}"
+	export RUST_SYSROOT="${STAGING_DIR_NATIVE}/${prefix_native}"
```

Comment:
Is this sysroot (pointing to native) always appropriate? Even if we're building for TARGET?

Also, it looks like this env var is used by rustc itself (the reset of these are specific to gcc-rs). Is there a reason it isn't simply exported in the rust.bbclass?